### ISSUE TITLE
Merge with other openvpn-radiusplugin repositories

### ISSUE THREAD

**Issue by alex-dot** [OpenVPN-Community/openvpn-radiusplugin#5]
I tried to compare the current source status of this repository with others out there, and either pull over the changes or at least the outstanding issues. Most could be integrated/moved.

Only two other repositories deviate strongly from this one, but being sourced from a far earlier (now defunct) repo:
https://github.com/cuckoohello/radiusplugin
https://github.com/PrivateCommunicationsCorp/radius-plugin

The difference is quite huge, but from a cursory glance mostly whitespace differences and formatting (which would still be quite welcome). Merging efforts will amount to a lot of chore-y labour, though.

**Comment by josephholsten** [comment]
I'll give then a look.

**Comment by josephholsten** [comment]
Well, since these aren't using a shared git origin, it's going to be especially non-trivial.

**Comment by josephholsten** [comment]
I tried merging PrivateCommunicationsCorp using rebase and format-patch/am, no dice. These are going to have to be manually merged. Also, I'm not sure I agree with the whitespace changes.

Once we get all the existing feature merges done, we should probably decide on a formatter and stick with it.

**Comment by alex-dot** [comment]
Yes, I would suggest looking at the functional differences (do they add functionality, fix bugs?) and, if any, port them is more sensible.

Agreed on the formatter. Seeing the diff I just realized that the current source is indeed quite inconsistent at times.

**Comment by sthibaul** [comment]
As mentioned on https://github.com/ValdikSS/openvpn-radiusplugin/issues/12 there are more of them:

* https://github.com/reee/radiusplugin
* https://github.com/squadette/radiusplugin
* https://github.com/comicchang/radiusplugin
* and more [https://github.com/search?utf8=✓&q=radiusplugin](https://github.com/search?utf8=%E2%9C%93&q=radiusplugin)
